cmd/kind/create/cluster/createcluster.go

```diff
@@ -50,6 +52,7 @@ func NewCommand() *cobra.Command {
 	cmd.Flags().StringVar(&flags.Config, "config", "", "path to a kind config file")
 	cmd.Flags().StringVar(&flags.ImageName, "image", "", "node docker image to use for booting the cluster")
 	cmd.Flags().BoolVar(&flags.Retain, "retain", false, "retain nodes for debugging when cluster creation fails")
+	cmd.Flags().DurationVar(&flags.Wait, "wait", time.Duration(0), "Wait for master node to be ready")
```

Comment:
`master` -> `control plane`
needs examples such `30s`, `10m`

Reply:
Added a default in the cmd line flag help message which indicates the use of units.
Other than that, to clarify this I added it in the user guide to avoid overcrowding the help message.